client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-stacked-features.tsx

```diff
@@ -50,4 +50,28 @@ const EmailProviderStackedFeatures: FunctionComponent< EmailProviderStackedFeatu
 	);
 };
 
-export default EmailProviderStackedFeatures;
+interface EmailProviderStackedFeaturesToggleButton {
+	handleClick: MouseEventHandler< HTMLButtonElement >;
+	isRelatedContentExpanded: boolean;
+}
+
+export const EmailProviderStackedFeaturesToggleButton: FunctionComponent< EmailProviderStackedFeaturesToggleButton > = (
+	props
+) => {
+	const translate = useTranslate();
+	const { handleClick, isRelatedContentExpanded } = props;
```

Comment:
Maybe we could move those props inside in the function parameters?

```suggestion
export const EmailProviderStackedFeaturesToggleButton: FunctionComponent< EmailProviderStackedFeaturesToggleButton > = (
	handleClick,
	isRelatedContentExpanded,
) => {
	const translate = useTranslate();
```

Reply:
I like it 👍